### DISCUSSION TITLE
How to reproduce ReProver training offline?

### DISCUSSION THREAD

**Discussion by laetitia-teo** [lean-dojo/ReProver#41] [Q&A]
I'm on a cluster on which the nodes are not directly connected to the internet. I'm trying to run the following command, from the ReProver section:

```bash
(ReProver) bash-4.4$ python generator/main.py fit --config generator/confs/cli_lean3_random.yaml
```

But when importing the leandojo module, the latter tries to access github (`LEAN4_REPO = GITHUB.get_repo("leanprover/lean4")`, line 76 of `src/leandojo/constants/py`); which fails with an error on my nodes. Any guidance on how to run this offline?

**Comment by yangky11** [comment]
LeanDojo may support it after https://github.com/lean-dojo/LeanDojo/pull/179 is merged.
